neural_lam/models/ar_model.py

```diff
@@ -508,10 +511,16 @@ def aggregate_and_plot_metrics(self, metrics_dict, prefix):
             )  # (N_eval, pred_steps, d_f)
 
             if self.trainer.is_global_zero:
+                metric_tensor_averaged = torch.mean(metric_tensor, dim=0)
+                # (pred_steps, d_f)
+
+                # Take square root after all averaging to change MSE to RMSE
+                if "mse" in metric_name:
```

Comment:
Here I wonder whether rmse should always **replace** mse. There might be a verification scenario where we want mse directly, no?

Reply:
I agree that it is probably not great to always get rid of MSE, if we would be interested in that. My thought process here was that this was the simplest implementation and one can still get MSE from just squaring the saved RMSE values.

Do you think it would be better to log both MSE and RMSE instead of replacing the MSE logging? That should be a simple change.

Reply:
You are right, simply squaring the rmse is easy enough and avoids duplicating the loss tensor. I would go ahead and merge the PR as is.